internal/session/query.go

```diff
@@ -10,15 +10,15 @@ const (
 insert into session_state
 with not_active as (
 	select session_id, 'active' as state
-	from 
```

Comment:
Do we want all these space diffs- there's a lot in this file.

Reply:
I can rip it out, I put this as a separate commit since I was conflicted on making the change. Generally I don't like seeing trailing whitespace, and `gofmt` removes them from go code, but since this sql code is in a string, they don't get removed automatically.